test/minting/error.txt

```diff
@@ -0,0 +1,140 @@
+  1) Contract: FiatToken_MiscTests
```

Comment:
We don't want to commit this file, do we?

Reply:
ooops - this is a personal scratch file.